docs/workforce-management-framework/topics/pro-configuring-logging.adoc

```diff
@@ -5,6 +5,9 @@ This section includes:
 
 //. xref:{context}-cloning-the-git-repository[Cloning the Git Repository]
 
+ifdef::upstream[]
```

Comment:
Thanks for that! That's really resolving the issue! 
Looks good, but curious if we really need to differentiate between upstream and non upstream version? 
It will be just easier to enforce just `discrete` format - We can move that out to modular docs and do some more granular tocs for portal. I have no strong opinion about that.